### PR TITLE
Feat: Added List Component

### PR DESCRIPTION
**Pull request by AshwiniSetty**
Added new Feat: List Component
This page contains,
1. Unordered List
2. Ordered List
3. Stacked List

Preview here:
https://deploy-preview-22--miniwii-ui.netlify.app/components/list/list

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
- Everything looks good to me.
- Clean code and quite understandable class names

**Review by dev-enforced — APPROVED** [review]
The live preview looks good and well documented about the list component  : https://deploy-preview-22--miniwii-ui.netlify.app/components/list/list
Naming pattern used for the stacked list and other provided lists also looks good and easy to understand.

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    font-size: 1.2rem;
+}
+
+.list-stacked li:hover {
+    background-color: var(--secondary-bg-color);
+}
+
+.list-stacked .list-item{
+    padding: 0.5rem;
+    font-size: 1rem;
```
Comment (Mohitk391) [inline components/list/list.css]: Suggestion: you can remove "font-size:1rem" as it is 1rem by default, so no need to explicitly provide that

**Review by Mohitk391 — APPROVED** [review]
Appreciation: You have written clean code.
One thing i found great is that you have kept css to minimum and not added unnecessary lines of code...The code looks clean and good to merge..

**Review by lomesshh — APPROVED** [review]
Clean code 
looks good to merge

File: components/list/list.css (hunk 1 above)
Comment (AshwiniSetty) [reply components/list/list.css]: Yes, @Mohitk391  thank you for that suggestion will change that.

**Pull request merged**